openstates/ri/legislators.py

```diff
@@ -98,6 +92,19 @@ def scrape(self, chamber, term):
                 "town_represented": d['town_represented'],
             }
 
+            if chamber == 'upper':
+                contact_url = 'http://webserver.rilin.state.ri.us/Email/SenEmailListDistrict.asp'            
+            if chamber == 'lower':
+                contact_url = 'http://webserver.rilin.state.ri.us/Email/RepEmailListDistrict.asp'
+            contact = lxml.html.fromstring(self.get(contact_url).text)
```

Comment:
self.lxmlize can be used instead here. It's a function we wrote (you can see the code here if you're interested: https://github.com/sunlightlabs/openstates/blob/master/openstates/utils/lxmlize.py)